tools/generators/ethereum/command_template_content.go

```diff
@@ -151,16 +160,22 @@ func {{$contract.ShortVar}}{{$method.CapsName}}(c *cobra.Command, args []string)
         return err
     }
 
-    {{ range $i, $param := .ParamInfos }}
-    {{$param.Name}}, err := {{$param.ParsingFn}}(args[{{$i}}])
-    if err != nil {
+       {{ range $i, $param := .CmdArgInfos }}
```

Comment:
This indentation is wrong as well (will get regenerated from `.tmpl` file), just leaving this comment to check the fix.